tests/device_tests/bitcoin/test_signtx_invalid_path.py

```diff
@@ -70,25 +55,24 @@ def test_invalid_path_fail(client: Client):
     )
 
     with pytest.raises(TrezorFailure) as exc:
-        btc.sign_tx(client, "Litecoin", [inp1], [out1], prev_txes=TX_CACHE_MAINNET)
+        btc.sign_tx(client, "Litecoin", [inp1], [out1], prev_txes={prev_hash: prev_tx})
 
     assert exc.value.code == messages.FailureType.DataError
     assert exc.value.message.endswith("Forbidden key path")
 
 
-# Adapted from TestMsgSigntx.test_one_one_fee,
-# only changed the coin from Bitcoin to Litecoin and set safety checks to prompt.
 # Litecoin does not have strong replay protection using SIGHASH_FORKID, but
 # spending from Bitcoin path should pass with safety checks set to prompt.
 @pytest.mark.altcoin
 def test_invalid_path_prompt(client: Client):
-    # tx: d5f65ee80147b4bcc70b75e4bbf2d7382021b871bd8867ef8fa525ef50864882
-    # input 0: 0.0039 BTC
+    address_n = parse_path("m/44h/0h/0h/0/0")
+    address = btc.get_address(client, "Bitcoin", address_n)
+    prev_hash, prev_tx = forge_prevtx([(address, 390_000)])
```

Comment:
In this file we seem to have this same setup three times - we could create a global `PREV_TXES = {PREV_HASH: PREV_TX}` as in some other places

Reply:
done in 6abaf09